### PR TITLE
Added fix for react warnings

### PR DESCRIPTION
**Pull request by ihor-yamshchykov**
If you initially pass undefined or null as the value prop, the component starts life as an "uncontrolled" component. Once you interact with the component, we set a value and react changes it to a "controlled" component, and issues the warning.

Changed initial values to empty strings instead of undefined.

### CODE REVIEW

File: src/inputTypes/textInput.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ class TextInput extends React.Component {
   }
 
   handleChange(e) {
+    console.log('here input');
```
Comment (bkozera) [inline src/inputTypes/textInput.js]: oh oh

File: src/inputTypes/textareaInput.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ class TextareaInput extends React.Component {
   }
 
   handleChange(e) {
+    console.log('here textarea')
```
Comment (bkozera) [inline src/inputTypes/textareaInput.js]: oh oh

File: dist/winterfell.min.js [hunk 3]
```diff
@@ -1,7 +1,12 @@
...==g?a[i]=g[i]:a[i]=t[i])}var y=arguments.length-2;if(1===y)a.children=n;else if(y>1){for(var _=Array(y),b=0;b<y;b++)_[b]=arguments[b+2];a.children=_}return m(e.type,u,s,p,d,h,a)},m.isValidElement=function(e){return"object"==typeof e&&null!==e&&e.$$typeof===h},m.REACT_ELEMENT_TYPE=h,e.exports=m}).call(t,n(1))},function(e,t,n){"use strict";function r(e){if(m){var t=e.node,n=e.children;if(n.length)for(var r=0;r<n.length;r++)g(t,n[r],null);else null!=e.html?p(t,e.html):null!=e.text&&f(t,e.text)}}function o(e,t){e.parentNode.replaceChild(t.node,e),r(t)}function i(e,t){m?e.children.push(t):e.node.appendChild(t.node)}function a(e,t){m?e.html=t:p(e.node,t)}function u(e,t){m?e.text=t:f(e.node,t)}function s(){return this.node.nodeName}function l(e){return{node:e,children:[],html:null,text:null,toString:s}}var c=n(38),p=n(35),d=n(48),f=n(84),h=1,v=11,m="undefined"!=typeof document&&"number"==typeof document.documentMode||"undefined"!=typeof navigator&&"string"==typeof navigator.userAgent&&/\bEdge\/\d/.test(navigator.userAgent),g=d(function(e,t,n){t.node.nodeType===v||t.node.nodeType===h&&"object"===t.node.nodeName.toLowerCase()&&(null==t.node.namespaceURI||t.node.namespaceURI===c.html)?(r(t),
```
Comment (bkozera) [inline dist/winterfell.min.js]: hmm why is this file shorter? and why is it in repo anyway?

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,4 @@
 .DS_Store
+.idea
```
Comment (bkozera) [inline .gitignore]: what's that?

File: src/inputTypes/textInput.js (hunk 1 above)
Comment (dmalts) [inline src/inputTypes/textInput.js]: console log

File: dist/inputTypes/textareaInput.js [hunk 5]
```diff
@@ -26,6 +26,7 @@ var TextareaInput = (function (_React$Component) {
   _createClass(TextareaInput, [{
     key: 'handleChange',
     value: function handleChange(e) {
+      console.log('here textarea');
```
Comment (dmalts) [inline dist/inputTypes/textareaInput.js]: log

File: dist/inputTypes/textInput.js [hunk 6]
```diff
@@ -26,6 +26,7 @@ var TextInput = (function (_React$Component) {
   _createClass(TextInput, [{
     key: 'handleChange',
     value: function handleChange(e) {
+      console.log('here input');
```
Comment (dmalts) [inline dist/inputTypes/textInput.js]: log

File: .gitignore (hunk 4 above)
Comment (ihor-yamshchykov) [inline .gitignore]: webstorm

**Pull request closed without merging**